plugins/reolink/src/reolink-api.ts

```diff
@@ -89,8 +89,12 @@ export class ReolinkCameraClient {
         await this.login();
         const url = options.url as URL;
         const params = url.searchParams;
-        for (const [k, v] of Object.entries(this.parameters)) {
-            params.set(k, v);
+        if(this.forceToken) {
```

Comment:
this change is not necessary.

Reply:
done, some calls fail if providing both token and user/pass, but if forceToken is set they will never be set

Reply:
How would both token and username pass be set?

Reply:
in this case it's not possible, it was something from the previous branch